farmer_registry.csv

```diff
@@ -0,0 +1,6 @@
+id,fullName,mobileNumber,dateOfBirth,gender,state,district,villageOrTown,postalCode,landArea,landOwnershipType,primaryCropType,secondaryCropType,face,farmerID
```

Comment:
this file can be removed, since CSV plugin will be covered as part of the docker compose setup

Reply:
File removed as csv plugin usecase will be covered as part of docker-compose